docker-build/Dockerfile.centos-6

```diff
@@ -0,0 +1,15 @@
+FROM centos:6
+
+# hadolint ignore=DL3033
+# see https://www.getpagespeed.com/server-setup/how-to-fix-yum-after-centos-6-went-eol
+RUN curl https://www.getpagespeed.com/files/centos6-eol.repo --output /etc/yum.repos.d/CentOS-Base.repo
```

Comment:
Please add the contents of the file to our repo, rather than downloading it. This ensures we can still run the workflow, even when getpagespeed.com goes down or vanishes.